map/apply-scheduled-updates-to-preplanned-map-area/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
```

Comment:
like we discussed today, this build.gradle is missing the necessary components to download the data, thanks for working to bring those in!